apps/tests/aws-runtime/test/test-service.ts

```diff
@@ -561,19 +564,20 @@ export const counterWatcher = counter.stream(
     // TODO: compute the possible operations union from the operations array
     if (item.operation === "remove") {
       const { n } = item.oldValue!;
-      await entitySignal2.sendSignal(item.key, { n: n + 1 });
+      await entitySignal2.sendSignal(item.key.id, { n: n + 1 });
     }
   }
 );
 
 export const counterNamespaceWatcher = counter.stream(
   "counterNamespaceWatch",
-  { namespacePrefixes: ["different"] },
+  { partitionPrefixes: ["different"] },
```

Comment:
What does this mean?

Reply:
It maps to the ability to filter on partial partition names for a stream. `[pk_name]: [{ prefix: ... }]`